jwst/resample/tests/test_resample_step.py

```diff
@@ -87,29 +82,40 @@ def nirspec_rate():
         'nints': 1,
         'nresets_between_ints': 0,
         'nsamples': 1,
-        'readpatt': 'NRSRAPID',
+        'readpatt': 'FAST',
         'sample_time': 10.0,
         'start_time': 58119.8333,
-        'type': 'NRS_FIXEDSLIT',
+        'type': 'MIR_LRS-SLITLESS',
         'zero_frame': False}
+    yield im
+    im.close()
```

Comment:
I haven't seen this yield-then-close() usage before - what does it do and/or how does it work, compared to a return statement?

Reply:
The yield pauses execution in the current function, returns a value to the caller, then waits for the function to do something with the value.  When the value goes out of scope, the function resumes processing.  I'm using it in these test fixtures to make sure that the created model gets closed after the test finishes using it.